ui: Adjust the size of custom chart to match regular charts

sankt-petersbug opened the pull request:
Unlike regular charts which are wrapped within a section and a flex
container, custom chart is placed directly under a section. This leads
to a noticable size(width) difference between reguar charts and custom
chart.

This fixes that by wrapping custom chart with a flex container.

Resolves: #27550

Release note (admin ui change): Make sure that custom chart and regular
charts have the same width.

- Before:

<img width="1382" alt="screen shot 2018-09-11 at 12 09 33 pm" src="https://user-images.githubusercontent.com/39347736/45377467-4d4c6280-b5c0-11e8-8e84-ea318b637130.png">

- After:
<img width="1381" alt="screen shot 2018-09-11 at 12 10 14 pm" src="https://user-images.githubusercontent.com/39347736/45377478-53424380-b5c0-11e8-840f-1e58fb54588f.png">

sankt-petersbug committed:
ui: Adjust the size of custom chart to match regular charts

Unlike regular charts which are wrapped within a section and a flex
container, custom chart is placed directly under a section. This leads
to a noticable size(width) difference between reguar charts and custom
chart.

This fixes that by wrapping custom chart with a flex container.

Resolves: #27550

Release note (admin ui change): Make sure that custom chart and regular
charts have the same width.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/30075)
<!-- Reviewable:end -->

couchand commented:
@sankt-petersbug Thanks for the contribution!  This looks good generally.

Could you please add screenshots to the PR description for before and after this change?  Thanks.

sankt-petersbug committed:
ui: Remove additional Section in Custom Chart when metric is empty

Because of the changes introduced by
93e5b40d20bdfa2f9f59a29eb8430fc312428f75, renderChart() suppose to
return a div element but it returns a section element when metric is
empty. This affects the styles of the "Empty Metric" message element.

The Section element is replaced with a div to maintain a consistent(unchanged) look.

Release note: None

sankt-petersbug commented:
@couchand Thank you for the quick feedback! 

I've added the screenshots to the PR description as you have requested. Also, I've pushed another commit to resolve an issue with having additional Section element when metric is empty. Please review that one two, and let me know if you have any questions or feedback.

couchand commented:
LGTM, thanks!

bors r+
